Features/Transfer/Sources/Services/AmountService.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import Primitives
+import PriceService
+import BalanceService
+import StakeService
+
+public struct AmountService {
```

Comment:
`AmountService` naming is a bit confusing, mb it would be better to name it `AssetDataService`

Also the `getRecipientAddress` method doesn't seem to fit the responsibility of this service